### DISCUSSION TITLE
sprintf equivalent of Percentage as string

### DISCUSSION THREAD

**Discussion by Gounlaf** [dlemstra/Magick.NET#1445] [Help] [answered]
:wave: 

I would like to find the `sprintf` equivalent of the expression of a [Percentage as string](https://github.com/dlemstra/Magick.NET/blob/df3c8de17c429c9d8096b9d43532d9714681f4f6/src/Magick.NET.Core/Types/Percentage.cs#L202).

Even with the [C# documentation,](https://learn.microsoft.com/en-us/dotnet/standard/base-types/custom-numeric-format-strings#the-0-custom-specifier) I'm a bit lost. Any help you will be great.

Thanks.
Regards.

**Comment by dlemstra** [comment, marked as answer]
I think you want `%.2f` or `%.2g`? I assume you are talking about C/C++?

**Reply by Gounlaf** [reply]
> I assume you are talking about C/C++?

Yes

`%.2g` seems to be the good one. I will mark as answered soon.

**Reply by Gounlaf** [reply]
I had to use the %.2f version (where the decimal part is fixed to 2 characters length) and remove the last .00 myself (for whole numbers).
